docs/topics/developer-tools.rst

```diff
@@ -254,15 +254,40 @@ page-number greater than 10), we increment the ``page`` attribute
 and ``yield`` a new request, inserting the incremented page-number 
 into our ``url``. 
 
-You can see that with a few inspections in the `Network`-tool we 
+In more complex websites, it could be difficult to easily reproduce the
+requests, as we could need to add ``headers`` or ``cookies`` to make it work. In those
+cases you can export the requests in `cURL <https://curl.haxx.se/>`_ format, by
+right-clicking on it in the network tool and using the
+:meth:`~scrapy.http.Request.from_curl()` method to generate an equivalent
+request. To do so, just import the :class:`~scrapy.http.Request` class and use this
+method to get the desired request::
+
+    from scrapy import Request
+
+    request = Request.from_curl(
+        "curl 'http://quotes.toscrape.com/api/quotes?page=1' -H 'User-Agent: Mozil"
+        "la/5.0 (X11; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0' -H 'Acce"
+        "pt: */*' -H 'Accept-Language: ca,en-US;q=0.7,en;q=0.3' --compressed -H 'X"
+        "-Requested-With: XMLHttpRequest' -H 'Proxy-Authorization: Basic QFRLLTAzM"
+        "zEwZTAxLTk5MWUtNDFiNC1iZWRmLTJjNGI4M2ZiNDBmNDpAVEstMDMzMTBlMDEtOTkxZS00MW"
+        "I0LWJlZGYtMmM0YjgzZmI0MGY0' -H 'Connection: keep-alive' -H 'Referer: http"
+        "://quotes.toscrape.com/scroll' -H 'Cache-Control: max-age=0'")
+
+
```

Comment:
1 line break is enough between paragraphs. Also applies _after_ the following paragraph.